Workleap.DotNet.CodingStandards.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Workleap.DotNet.CodingStandards</id>
+    <version>$version$</version>
+    <authors>Workleap</authors>
+    <owners>Workleap</owners>
+    <developmentDependency>true</developmentDependency>
+    <license type="expression">Apache-2.0</license>
+    <readme>README.md</readme>
+    <projectUrl>https://github.com/gsoft-inc/wl-dotnet-codingstandards</projectUrl>
+    <description>Workleap recommended coding standards for .NET.</description>
+    <copyright>Copyright © Workleap</copyright>
+    <repository type="git" url="$RepositoryUrl$" commit="$RepositoryCommit$" branch="$RepositoryBranch$" />
+    <dependencies>
+      <dependency id="Microsoft.CodeAnalysis.BannedApiAnalyzers" version="3.3.4" />
```

Comment:
Have we considered adding other open-source analyzers?

Reply:
It has been agreed with consumers that for now we would only provide built-in .NET analyzers (style, perf, etc.) and banned APIs.

But we can definitely consider adding more later on (if it makes sense)